### PR TITLE
[STRAT-92] Getting speed limits from Bing Maps API

### PR DESCRIPTION
**Pull request by j985chen**
Created a new file called get_speedlimits.py under routemodel/data_retrieval that uses the Bing Maps Snap to Road API to get the speed limit between a list of points

### CODE REVIEW

File: routemodel/data_retrieval/get_speedlimits.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+sys.path.append(os.path.dirname(__file__))
+from config import API_KEY
+BASE_URL = "https://dev.virtualearth.net/REST/v1/Routes/"
+
+class SpeedlimitDataRetrieval: 
+    def __init__(self, pts: list, sp_unit="KPH"):
+        """
+        Initialize SpeedlimitDataRetrieval
+        @param pts: list containing latitude and longitude pairs of points along route
+        @param sp_unit: speed unit response data option
```
[the remark this answers is not in the document]
Comment (j985chen) [reply routemodel/data_retrieval/get_speedlimits.py:15]: MPH

**Review by e-wai — APPROVED** [review]
Looks good 🎉

File: routemodel/data_retrieval/get_speedlimits.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        response = requests.get(url)
+        return response
+
+    def parse_speedlimit_data(self, response): 
+        """
+        Parsing through Routes API call response.
+        @param response: Requests.response object from API call
+        @return: boolean
+        """
+
```
Comment (e-wai) [inline routemodel/data_retrieval/get_speedlimits.py:57]:
```suggestion
pass
```

**Review by e-wai — APPROVED** [review]
Sorry. Added a quick pass to the last method, but ready to merge

**Pull request merged**